### ISSUE TITLE
Blender 2.79 - Mouse Middle Button problem

### ISSUE THREAD

**Issue by AlonDan** [TrueCryer/blendermada_client#14]
I just installed the latest Blendermada addon to the fresh official Blender 2.79 (released today).
Everything works nice but only one issue:

The preview image is taking control (first priority) on everything on blender.
So if I hold the middle mouse button to pan on the overview it won't pan, but it will pan the Preview image of the Blendermada add-on.

Would be great if this will be fixed, this addon is fantastic! keep up the good job. :)

**Comment by TrueCryer** [comment]
Yes, it's not a problem of 2.79 release but all releases. Preview catches events from all Blender screen and I don't know how to fix it.
But, for fast working I set up left and right buttons to quick closing preview.

**Comment by AlonDan** [comment]
Thanks for the reply, I'll give it a shot. 
Keep up the good job!
